### ISSUE TITLE
[OJS] Modify submission date using QuickSubmit plugin

### ISSUE THREAD

**Issue by fmotoki** [pkp/pkp-lib#5418]
Hi! QuickSubmit for OJS3 no longer allows to change the submission date. Setting this date is crucial for newly migrated journals, as well as journals that use a separate editorial manager such as ScholarOne. It used to possible on OJS2, I cannot fathom why the functionality was dropped.

**Comment by asmecher** [comment]
@fmotoki, I don't think OJS 2.x and 3.x are different in this regard -- the submission date is always the date on which the submission gets entered into OJS (i.e. the date QuickSubmit is used to create the submission). The publication date can be entered on the Quick Submit form in both OJS 2.x and 3.x.

**Comment by fgnievinski** [comment]
indeed, in the quicksubmission plugin in OJS 2 there was no submission date field, only publication date:
![image](https://user-images.githubusercontent.com/5018500/81140229-0bafbe80-8f3f-11ea-835d-bcfd6b923e35.png)

now, earlier versions of OJS 3 had a minor bug in the data input form, which has already been fixed:
https://forum.pkp.sfu.ca/t/quicksubmit-plugin-publication-date/53341/4

in the forum there are discussions about setting the submission date, so maybe this bug report should be reframed as a feature request?
https://forum.pkp.sfu.ca/t/quick-submit-modify-submission-date/51567

here's an old related issue: 
https://github.com/pkp/pkp-lib/issues/1238
